main.html

```diff
@@ -1,77 +1,255 @@
+  Sure, it's only one single guest team submitting the change, but that team is likely representative of many other consumers and their needs.
+</p>
+
+<p>
+  In addition to this alignment, there is also a grass-roots evangelism that takes hold in many contributors.
+  Having made a code contribution, they feel a stake in the host team project.
+  It's something that they recommend to their colleagues or new teams that they join.
+  They understand the project better and are able to answer questions about it to others, relieving the host team of some of that burden.
+</p>
+
+<h2>Principles</h2>
+
+<p>
+  Every company, team, project, and individual is different.
+  Because of that fact, the exact way that the concept of InnerSource works will vary from one situation to another.
+  At its core, however, are two principles that form the bedrock of any successful instance of InnerSource.
+  These principles must be respected and present in order for InnerSource to achieve the benefits described earlier.
+  The principles are:
+</p>
+
+<ul>
+  <li>Voluntary Code Contribution</li>
+  <li>Prioritized Mentorship</li>
+</ul>
+
+<p>
+  Let's take a look at each of these principles in more detail.
+</p>
+
+<h3>Voluntary Code Contribution</h3>
+
+<p>
+  The first word <em>Voluntary</em> means that engagement in InnerSource from both the guest and host teams occurs of their own free will.
+  The guest team voluntarily donates code to the host team and the host team voluntarily accepts it.
+  This opt-in nature means that each team needs to be certain that their involvement adds value to the others' objectives.
+  Never is a host team required to accept a contribution that isn't in ultimate alignment with their overall mission.
+  Never is a guest team required to submit a contribution that doesn't ultimately further their own mission and priorities.
+</p>
+
+<p>
+  The word <em>Code</em> emphasizes that the collaboration between guest and host goes all the way to the code.
+  Guest involvement in opening issues, updating requirements, fixing docs, etc. is good, but collaboration needs to reach as far as submitting code to achieve all of the benefits that we've discussed.
+</p>
+
+<h3>Prioritized Mentorship</h3>
+
+<p>
+  <em>Mentorship</em> from host team to guest team is a key aspect of InnerSource.
+  Contributors on guest teams are upleveled so that they understand enough about the host team system to change it successfully.
+  In the process of doing so, they come to better understand the host team system as a general consumer.
+  They can use it more effectively and help other consumers to do so as well. 
+</p>
+
+<p>
+  It's critical that this mentorship for contributors is <em>Prioritized</em> by the host team.
+  Trusted committer availability to mentor is made available to guest team contributors <em>at the time that they need it</em> as opposed to when it is convenient for the trusted committer and host team.
```

Comment:
How does that work with async workflows?  `At the time`, implies immediate but in reality there is at least a 4-8 hour lag due to async workflows between team members and even team cadences.